src/components/Employee/Taxes/Taxes.tsx

```diff
@@ -57,19 +55,31 @@ const Root = (props: TaxesProps) => {
   const { onEvent, fieldErrors, baseSubmitHandler } = useBase()
   useI18n('Employee.Taxes')
 
-  const { data: employeeFederalTaxes } = useGetEmployeeFederalTaxes(employeeId)
-  const { data: employeeStateTaxes } = useGetEmployeeStateTaxes(employeeId)
+  const { data: fedData } = useEmployeeTaxSetupGetFederalTaxesSuspense({
+    employeeUuid: employeeId,
+  })
+  const employeeFederalTax = fedData.employeeFederalTax!
+
+  const { mutateAsync: updateFederalTaxes, isPending: isPendingFederalTaxes } =
+    useEmployeeTaxSetupUpdateFederalTaxesMutation()
+
+  const { data: stateData } = useEmployeeTaxSetupGetStateTaxesSuspense({
+    employeeUuid: employeeId,
+  })
+  const employeeStateTaxes = stateData.employeeStateTaxesList!
+  const { mutateAsync: updateStateTaxes, isPending: isPendingStateTaxes } =
+    useEmployeeTaxSetupUpdateStateTaxesMutation()
 
   const defaultValues = {
-    ...employeeFederalTaxes,
-    two_jobs: employeeFederalTaxes.two_jobs ? 'true' : 'false',
-    deductions: employeeFederalTaxes.deductions ? Number(employeeFederalTaxes.deductions) : 0,
-    dependents_amount: employeeFederalTaxes.dependents_amount
-      ? Number(employeeFederalTaxes.dependents_amount)
+    ...employeeFederalTax,
+    twoJobs: employeeFederalTax.twoJobs ? 'true' : 'false',
+    deductions: employeeFederalTax.deductions ? Number(employeeFederalTax.deductions) : 0,
+    dependentsAmount: employeeFederalTax.dependentsAmount
+      ? Number(employeeFederalTax.dependentsAmount)
       : 0,
-    other_income: employeeFederalTaxes.other_income ? Number(employeeFederalTaxes.other_income) : 0,
-    extra_withholding: employeeFederalTaxes.extra_withholding
-      ? Number(employeeFederalTaxes.extra_withholding)
+    otherIncome: employeeFederalTax.otherIncome ? Number(employeeFederalTax.otherIncome) : 0,
+    extraWithholding: employeeFederalTax.extraWithholding
+      ? Number(employeeFederalTax.extraWithholding)
```

Comment:
We'll need to be sure to keep BQE informed with the next release that default values have camel case moving forward since this is introducing a breaking change